examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -49,20 +49,23 @@ import("${silabs_common_plat_dir}/efr32/args.gni")
 
 # Sanity check
 assert(!(chip_enable_wifi && chip_enable_openthread))
-assert(!(use_rs911x && chip_enable_openthread))
+assert(!(use_rs9116 && chip_enable_openthread))
+assert(!(use_SiWx917 && chip_enable_openthread))
 assert(!(use_wf200 && chip_enable_openthread))
 
 if (chip_enable_wifi) {
-  assert(use_rs911x || use_wf200)
+  assert(use_rs9116 || use_wf200 || use_SiWx917)
   enable_openthread_cli = false
   import("${chip_root}/src/platform/silabs/efr32/wifi_args.gni")
 
-  if (use_rs911x) {
+  if (use_rs9116) {
     wiseconnect_sdk_root =
         "${chip_root}/third_party/silabs/wiseconnect-wifi-bt-sdk"
     import("rs911x/rs911x.gni")
```

Comment:
so you have now two separated, wise SDK ?

Reply:
Yes @jmeg-sfy We are having two seperate SDK's. 
1) Wiseconnect SDK for rs911x.
2) Wisemcu SDK for 917 SOC and 917 NCP.